manual/src/html_processing/scss/manual.scss

```diff
@@ -293,7 +293,13 @@ div.caml-example.toplevel div.caml-input::before{
 span.number{
     padding-right: 1ex;
 }
-span.c004, span.c002{
+span.c004, span.c005 {
+	font-family: "Fira Mono", courier, monospace;
+}
+span.c003, span.c005 {
+	color: rgba(91, 33, 6, 0.87);
+}
+span.c002{
```

Comment:
What is the motivation for the color change on c0003, c0005 ? The font change?

Reply:
The font change for `.c004` and `.c005` sets respectively the `Match_failure` and `let` in the screenshot in monospace font. I tried to follow the CSS rules from the old HTML manual. I took the font-family from other parts of the file, but I added the `monospace` generic fallback at the end, as is good practice. Should I fix that in other places of the file while I'm here? 

 The color `#888888` that was used for (some) monospace fragments has simply too low contrast on `#ffffff`, fails [accessibility guidelines](https://webaim.org/resources/contrastchecker/) (even more on darker backgrounds like in tables) and looks grayed out and unimportant (to me, identifiers and syntax fragments are pretty important and should not blend into the background).  

Instead I chose a safe, non-offending color by simply darkening the color currently used for links (and adding a bit of transparency, to make it work better on darker backgrounds like in tables). Maybe a later PR will pick something better, but I wouldn't want to discuss this choice now.

Reply:
I agree that switching the font for inline code to monospace is the right thing to do (we should get ride of those hevea generated classes at some points).
 
Adding a `monospace` failback to the CSS sounds like a good idea (in a separate commit probably).

I agree that the previous color has too little contrast. Having a color that is close from the grammatical symbol color seems like a sensible idea.